asistente_ladm_col/lib/db/pg_connector.py

```diff
@@ -274,41 +288,91 @@ def get_annex17_point_data(self, plot_id):
         query = annex17_point_data_query.get_annex17_point_data_query(self.names, self.schema, plot_id)
         cur.execute(query)
 
-        return cur.fetchone()[0]
+        return True, cur.fetchone()[0]
 
-    def get_ant_map_plot_data(self, plot_id, mode='only_id'):
+    def get_ant_map_plot_data(self, plot_id, mode, overview):
         res, msg = self.check_and_fix_connection()
         if not res:
             return (res, msg)
 
-        where_id = ""
-        if mode != 'all':
-            where_id = "WHERE {LC_PLOT_T}.{T_ID_F} {operation} {plot_id}".format(LC_PLOT_T=self.names.LC_PLOT_T,
-                                                                                 T_ID_F=self.names.T_ID_F,
-                                                                                 operation='=' if mode == 'only_id' else '!=',
-                                                                                 plot_id=plot_id)
+        if mode:
+            where_id = "WHERE {LC_PLOT_T}.{T_ID_F} = {plot_id}".format(LC_PLOT_T=self.names.LC_PLOT_T,
+                                                                       T_ID_F=self.names.T_ID_F,
+                                                                       plot_id=plot_id)
+        else:
+            scale_zoom = 1000 if overview else 100
+            where_id = """
+                        WHERE {LC_PLOT_T}.{LC_PLOT_T_GEOMETRY_F} &&
+                        (SELECT ST_Expand(ST_Envelope({LC_PLOT_T}.{LC_PLOT_T_GEOMETRY_F}), {scale_zoom})
+                        FROM {schema}.{LC_PLOT_T} WHERE t_id = {plot_id}) AND {LC_PLOT_T}.{T_ID_F} != {plot_id} 
+                       """.format(LC_PLOT_T_GEOMETRY_F=self.names.LC_PLOT_T_GEOMETRY_F,
+                                  LC_PLOT_T=self.names.LC_PLOT_T,
+                                  T_ID_F=self.names.T_ID_F,
+                                  plot_id=plot_id,
+                                  scale_zoom=scale_zoom,
+                                  schema=self.schema)
 
         cur = self.conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
-
         query = ant_map_plot_query.get_ant_map_query(self.names, self.schema, where_id)
         cur.execute(query)
 
-        if mode == 'only_id':
-            return cur.fetchone()[0]
+        if mode:
+            return True, cur.fetchone()[0]
         else:
-            return cur.fetchall()[0][0]
+            return True, cur.fetchall()[0][0]
 
-    def get_ant_map_neighbouring_change_data(self, plot_id):
+    def get_ant_map_road_nomenclature(self, plot_id, overview):
         res, msg = self.check_and_fix_connection()
         if not res:
-            return (res, msg)
+            return res, msg
+
+        if not self.cadastral_cartography_model_exists():
+            return False, 'Cadastral cartography model it is not implemented'
```

Comment:
"Cadastral cartography model is not implemented".

Regarding the message, do you mean "is not implemented" or "was not found in the database"?

Reply:
You're right, the correct message it something like that "Cadastral cartography model was not found in the database"

Reply:
Good.